src/web/web-service/wpan_service.cpp

```diff
@@ -383,6 +383,9 @@ std::string WpanService::HandleStatusRequest()
     VerifyOrExit((rval = client.Execute("partitionid")) != nullptr, ret = kWpanStatus_GetPropertyFailed);
     networkInfo["Network:PartitionID"] = rval;
 
+    VerifyOrExit((rval = client.Execute("pskc")) != nullptr, ret = kWpanStatus_GetPropertyFailed);
+    networkInfo["Network:PSKc"] = rval;
```

Comment:
We should not encourage directly displaying a static credential that can be used to administer the network.